codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -185,6 +185,7 @@ publicise->publicize
 publicised->publicized
 publicises->publicizes
 publicising->publicizing
+ractise->practice
```

Comment:
I don't think `ractise` is a real word, even in British English?

Reply:
`practise` is a real word and is the variant of `practice`but the British and Australian spelling of it. `ractise` is just a typo of `practise`

Reply:
yeah, but the dictionary_en-GB_to_en-US.txt wants  
valid Be->valid AE

you want to add the typo to the regular dictionary.